codebuild/codebuild.config

```diff
@@ -50,15 +50,10 @@ env: S2N_LIBCRYPTO=openssl-1.1.1 OPENSSL_ia32cap="~0x200000200000000" BUILD_S2N=
 snippet: UbuntuBoilerplate2XL
 env: S2N_LIBCRYPTO=boringssl BUILD_S2N=true TESTS=integration GCC_VERSION=9
 
-# Unit Test
-[CodeBuild:s2nUnitOpenSSL111Gcc6]
-snippet: UbuntuBoilerplateLarge
-env: S2N_LIBCRYPTO=openssl-1.1.1 BUILD_S2N=true TESTS=unit GCC_VERSION=6 S2N_COVERAGE=true CODECOV_IO_UPLOAD=true
```

Comment:
From offline chat: 

Currently we have asan tests running only on openssl version 1.0.2 and not on openssl version 1.1.1. This change prioritizes running asan on openssl-1.1.1. Ideally we would like to have both, an option to get both tasks running is to combine valgrind and asan tests in one job.